inst/templates/get-started.Rmd

```diff
@@ -383,6 +383,49 @@ vroom::vroom_write(to_edit, "to_edit.csv", delim = ",")
 writexl::write_xlsx(to_edit, "to_edit.xlsx")
 ```
 
+### Calculate coverage statistics
+
+```{r}
+# Dataframe with TRUE suggested matches
+suggested <- to_edit %>%
+  filter(suggest_match)
+
+# Matched companies based on suggested matches
+matched_rows <- suggested %>% nrow()
+
+# Share of matched companies from total loanbook companies
+matched_share <- matched_rows / loanbook %>%
+  distinct(company_name) %>%
+  nrow()
+matched_share
+
+# Number and share of matched companies classified by misc_info from total
+# loanbook companies
+# misc_info should be replaced with variables like sectors, headcount etc.
+x <- loanbook %>% count(misc_info)
+y <- suggested %>% count(misc_info)
+misc_share <-
+  left_join(x, y, by = c("misc_info"), suffix = c("_loanbook", "_merged")) %>%
+  mutate(n_share = n_merged / n_loanbook)
+
+# Calculate loan exposure categorised based on misc_info id_tilt column is used
+# as the substitute due to unavailability of loan amount column in sample data.
+# Please replace id_tilt column with the loan amount column.
```

Comment:
This comment seems like a better fit for being just prose --- rather than a code comment. How about breaking the code chunk above and below, then leaving this comment as prose?

```r
# Some code
...
```

Some prose.

```r
# Some more code
...
```


The same may be true for other comments.